### DISCUSSION TITLE
Bug in my feed

### DISCUSSION THREAD

**Discussion by Nergahak** [community/community#137902] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

My feed has a bug, I configured it to only display "releases" and "stars", however earlier today the information appeared normally, now, no information appears, not even after F5, simply nothing else happens. I also know that I'm wasting my time coming here to post this, because since Github ended direct support, this place has become a mess.

**Comment by joemar25** [comment]
Can you try clearing your browser cache and reloading the page?

Sometimes, that can resolve the problem. Also, double-check your feed settings to make sure they're still configured to display "releases" and "stars". If the issue persists, try switching to a different browser to see if the problem is browser-specific. If none of these steps resolve the issue, please provide more details about your feed configuration and any error messages you might be seeing.

**Comment by Nergahak** [comment]
I did it like this, I went to Github, my feed didn't load, I cleared my Firefox cache and reloaded the page and it didn't work either. So I accessed my feed settings and without doing anything, I just clicked on "save" and only after doing that, the feed loaded, but remember, I didn't do anything. I hope I don't have to do this all the time and no, I won't test it in another browser, because I use Firefox and if it doesn't work in it, it won't work in another either. 

I can see that this won't have any practical results, so I'll end it here.

EDIT: I'm closing this topic, not because this problem has been solved, because it hasn't, but because I've given up.

**Reply by joemar25** [reply]
ok, Its fine to give up.. Goodluck to your journey sir..

**Reply by Nergahak** [reply]
Don't get me wrong, I didn't mean to be rude, I am angry, but with Github. By the way, I'm a girl (biologic).

**Reply by joemar25** [reply]
Sorry to hear that you're frustrated with Github, and I apologize if my response came across as unhelpful.

**Comment by Blahx** [comment]
I seem to have the same problem suddenly.

I was looking at my feed and checking through what releases I missed, refreshed and suddenly all releases disappeared. it has been a day since and still nothing.

**Reply by Blahx** [reply]
https://github.com/orgs/community/discussions/122324#discussioncomment-10576348

@Nergahak This fixed it for me

**Reply by Nergahak** [reply]
Nothing described in this link works, because it was already like that.

**Comment by Nergahak** [comment, marked as answer]
I'm going to mark it as answered to definitively close this topic, unfortunately Github technicians will only give me attention if I pay for it.

**Comment by llussy** [comment]
I also encountered the same problem.
